src/reachy2_sdk/config/reachy_info.py

```diff
@@ -10,15 +11,22 @@
 
 
 class ReachyInfo:
-    """The ReachyInfo class saves information of the robot that won't be modified during the session.
-
-    The ReachyInfo class gives access to:
-        - the robot's serial_number, that will never change
-        - the robot's hardware version, that will not change during a session
-        - the robot's core software version, that will not change during a session
+    """The ReachyInfo class saves information of the global robot.
+
+    The ReachyInfo class gives access to informations that won't be modified during the session:
+        - the robot's hardware version
```

Comment:
if you delete the space before the "-", it displays a real list instead of a one line separated by dash